frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -347,33 +347,12 @@ const ManageSoftwarePage = ({
       >
         {`${count} software item${count === 1 ? "" : "s"}`}
         <span className="count-last-updated">
-          {`Last updated ${lastUpdatedAt}`}{" "}
-          <span className={`tooltip`}>
-            <span
-              className={`tooltip__tooltip-icon`}
-              data-tip
-              data-for="last-updated-tooltip"
-              data-tip-disable={false}
-            >
-              <img alt="question icon" src={QuestionIcon} />
-            </span>
-            <ReactTooltip
-              place="top"
-              type="dark"
-              effect="solid"
-              backgroundColor="#3e4771"
-              id="last-updated-tooltip"
-              data-html
-            >
-              <span className={`tooltip__tooltip-text`}>
-                Fleet periodically
-                <br />
-                queries all hosts
-                <br />
-                to retrieve software
-              </span>
-            </ReactTooltip>
-          </span>
+          <TooltipWrapper
+            tipContent="Fleet periodically queries all hosts to retrieve software"
+            position="top"
+          >
+            {`Last updated ${lastUpdatedAt}`}
```

Comment:
Should we go ahead and migrate this to use the `LastUpdatedText` that you created?

Reply:
Great eye!